amundsen_application/static/js/components/SearchPage/index.tsx

```diff
@@ -167,63 +198,69 @@ class SearchPage extends React.Component<SearchPageProps, SearchPageState> {
       )
     }
 
-    const items = this.props.tables;
-    const { page_index, results, total_results } = items;
-    const { popularTables } = this.props;
-
-    const showResultsList = results.length > 0 || popularTables.length > 0;
-
-    if (showResultsList) {
-      const startIndex = (RESULTS_PER_PAGE * page_index) + 1;
-      const endIndex = RESULTS_PER_PAGE * ( page_index + 1);
-      let listTitle = `${startIndex}-${Math.min(endIndex, total_results)} of ${total_results} results`;
-      let infoText = "Ordered by the relevance of matches within a resource's metadata, as well as overall usage.";
-      const searchListParams = {
-        source: 'search_results',
-        paginationStartIndex: RESULTS_PER_PAGE * page_index
-      };
-
-      const showPopularTables = total_results < 1;
-      if (showPopularTables) {
-        listTitle = 'Popular Tables';
-        infoText = "These are some of the most commonly accessed tables within your organization.";
-        searchListParams.source = 'popular_tables';
-      }
+    const tabConfig = [
+      {
+        title: `Tables (${ this.props.tables.total_results })`,
+        key: ResourceType.table,
+        content: this.getTabContent(this.props.tables),
+      },
+      {
+        title: `Users (${ this.props.users.total_results })`,
+        key: ResourceType.user,
+        content: this.getTabContent(this.props.users),
+      },
+    ];
 
-      return (
-        <div className="col-xs-12">
-          <div className="search-list-container">
-            <div className="search-list-header">
-              <label> { listTitle } </label>
-              <InfoButton infoText={ infoText }/>
-            </div>
-            <SearchList results={ showPopularTables ? this.props.users.results : results } params={ searchListParams }/>
-          </div>
-          <div className="search-pagination-component">
+    return (
+      <div className="col-xs-12">
+        <TabsComponent
+          tabs={ tabConfig }
+          defaultTab={ ResourceType.table }
+          activeKey={ this.state.selectedTab }
+          onSelect={ this.onTabChange }
+        />
+      </div>
+    );
+  };
+
+
+  // TODO: Hard-coded text strings should be translatable/customizable
+  getTabContent = (results) => {
+    const startIndex = (RESULTS_PER_PAGE * results.page_index) + 1;
```

Comment:
Perhaps I've gotten lost in the diffs, but how does the UI currently handle a search that returns no results for the tab, or if the given index is out of bounds (possible with someone typing in/modifying a url to form their query)

Reply:
I'll need to modify `createErrorMessage` to accept a custom string for the tab type and call it in this function. I put a TODO for now, but it should be easy to fix.